tasks/logs.py

```diff
@@ -0,0 +1,63 @@
+"""
+Logs tasks
+"""
+import os
+import shutil
+from distutils.dir_util import copy_tree
+
+import invoke
+from invoke import task
+from invoke.exceptions import Exit
+
+from .utils import bin_name
+from .utils import REPO_PATH
+
+LOGS_BIN_PATH = os.path.join(".", "bin", "logs")
+LOGS_BIN_NAME = os.path.join(LOGS_BIN_PATH, bin_name("logs"))
+LOGS_DIST_PATH = os.path.join(LOGS_BIN_PATH, "dist")
+
+@task
+def build(ctx):
+    """
+    Build Logs Agent
+    """    
+    cmd = "go build -o {bin_name} {REPO_PATH}/cmd/logs/"
```

Comment:
you also need to specify the build tags when you use `go build`, otherwise the packages you're using from `datadog-agent/pkg` (outside of `logs`) will not include all the features they can offer.

Here, unless I'm missing something, I think you could simply use the list defined at `.build_tags.ALL_TAGS`

Reply:
We were only using `docker` but I removed the dependency as we were only using `ContainerIDToEntityName`.